server/service/src/sync/translations/requisition.rs

```diff
@@ -202,6 +209,9 @@ impl SyncTranslation for RequisitionTranslation {
             min_months_of_stock: data.thresholdMOS,
             linked_requisition_id: data.linked_requisition_id,
             expected_delivery_date: data.expected_delivery_date,
+            program_id: data.programID.clone(),
+            period_id: data.periodID,
+            order_type_id: find_order_type_id(conn, data.programID, data.orderType)?,
```

Comment:
Sorry I don't think I described it clearly. I thought that orderType on requisition should just be the `name` of the orderType, since we can change program settings in mSupply in a way where order type is 'deleted'.  Just copying over properties of the order type seemed like the only way to do this (although there are other side effects that may be problematic when order type name is change). What do you think ?

Reply:
This is actually what you said, but I assumed it was a typo given the fact that we have an order type table now.
Your suggestion does simplify things though, and if it's consistent with mSupply I'm fine it.

Like you say it depends what behaviour we want. If we change the name of an order type should it affect old requisitions. I think there's a good argument to say that it should show as it was when created.